pkg/virtual/framework/fixedgvs/apiserver/apiserver.go

```diff
@@ -118,11 +118,11 @@ func (c completedConfig) New(virtualWorkspaceName string, groupManager discovery
 
 	storage := map[string]restStorage.Storage{}
 	for resource, storageBuilder := range c.ExtraConfig.StorageBuilders {
-		restStorage, err := storageBuilder(c.GenericConfig)
```

Comment:
revert this one please

Reply:
what is about `rs` alias for the imported package?